cypress/support/index.js

```diff
@@ -75,3 +78,25 @@ if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
     cy.stopDummyServer();
   });
 }
+
+/**
+ * Make setup step in here so that all the test files in dashboards-assistant
+ * won't need to call these commands.
+ */
+if (
+  Cypress.env('DASHBOARDS_ASSISTANT_ENABLED') &&
+  Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')
+) {
+  const originalBackendEndpoint = currentBackendEndpoint.get();
+  before(() => {
+    currentBackendEndpoint.set(currentBackendEndpoint.REMOTE_NO_AUTH);
+    cy.addAssistantRequiredSettings();
+    cy.readOrRegisterRootAgent();
+    cy.startDummyServer();
+  });
+  after(() => {
+    currentBackendEndpoint.set(originalBackendEndpoint, false);
```

Comment:
Should call `currentBackendEndpoint.set(currentBackendEndpoint.REMOTE_NO_AUTH);` here.